Src/ILGPU/IR/Analyses/Loops.cs

```diff
@@ -740,7 +740,7 @@ private Loops(CFG<TOrder, TDirection> cfg)
             {
                 foreach (var predecessor in member.GetPredecessors<TDirection>())
                 {
-                    if (!nodeMapping[predecessor].IsInSCC && 
+                    if (!nodeMapping[predecessor].IsInSCC &&
```

Comment:
Suggestion: Maybe we can skip this change for now since it might make more sense to include this tiny update in the loop unrolling PR. What do you think?

Reply:
Rebased successfully now that the loop unrolling PR in merged.